### ISSUE TITLE
Improve instructions for form sharing

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#1913]
Based on usability testing, the instructions for form sharing could be improved.

Notes from testing:

**Does sharing an attached JSON file through the app work for users?**
Yes, the instructions on sharing could be improved slightly, for the most part they were clear and users had no trouble understanding what to do. The only confusion noted was that the two options, to create an account, or to download the file were not distinct enough.

**Comment by Abi-Nada** [comment]
@connorscarolyns do you think the current instructions we have in the application now satisfies this issue? Or would you like this card to be open to revisit at a later time?

**Comment by connorscarolyns** [comment]
No, the instructions are still the same as when tested; there could be some modifications to improve clarity

**Comment by anikbrazeau** [comment]
Addressed here: https://articles.alpha.canada.ca/forms-formulaires/collaborating-in-gc-forms/
